website/.prettierrc.js

```diff
@@ -2,4 +2,13 @@
 
 module.exports = {
   singleQuote: true,
+  overrides: [
+    {
+      files: '*.hbs',
+      options: {
+        singleQuote: false,
```

Comment:
This isn't being followed in the .hbs below, something is odd (I looked at this briefly on Friday and then was end of the day). I'll reopen the issue so don't forget